include/verilated_fst_c.cpp

```diff
@@ -77,10 +77,16 @@ VerilatedFst::VerilatedFst(void* fst)
     : m_fst(fst)
     , m_fullDump(true)
     , m_nextCode(1)
-    , m_scopeEscape('.') {
+    , m_scopeEscape('.')
+    , m_symbolp(NULL) {
     m_valueStrBuffer.reserve(64 + 1);  // Need enough room for quad
 }
 
+VerilatedFst::~VerilatedFst() {
+    if (m_fst == NULL) { fstWriterClose(m_fst); }
```

Comment:
[Meant to be part of above]

I think this should be if(m_fst) fstWriterClose(m_fst);

As written I don't see how it makes sense. I understand you just moved this, but might as well try to clean up....